schema/column.go

```diff
@@ -26,10 +30,11 @@ type Column struct {
 	Type ValueType `json:"type,omitempty"`
 	// Description about column, this description is added as a comment in the database
 	Description string `json:"-"`
-	// Column Resolver allows to set you own data based on resolving this can be an API call or setting multiple embedded values etc'
+	// Column Resolver allows to set your own data for a column; this can be an API call, setting multiple embedded values, etc
 	Resolver ColumnResolver `json:"-"`
 	// Creation options allow modifying how column is defined when table is created
-	CreationOptions ColumnCreationOptions `json:"creation_options,omitempty"`
+	CreationOptions  ColumnCreationOptions `json:"creation_options,omitempty"`
+	IsIncrementalKey bool                  `json:"is_incremental_key,omitempty"`
```

Comment:
Missing godoc (all the other fields have explanations)

Reply:
Nice catch, I think this field got added here by accident actually